### ISSUE TITLE
Reduce the verbosity install and update commands

### ISSUE THREAD

**Issue by zabil** [getgauge/gauge#712]
Reduce the verbosity of install and update command and show it only when the `--log-level` is set to `debug` or on failures.

### Expected behavior
```
✔︎ ~ gauge install ruby
.......................
Successfully installed ruby plugin
```
### Actual behavior
```
✔︎ ~ gauge install ruby
Gathering metadata for ruby

Downloading gauge-ruby-0.4.1-linux.x86_64.zip
.......................
Installing plugin ruby 0.4.1
Successfully installed plugin 'ruby'.
```
### Gauge version
```
Gauge version: 0.9.0.nightly-2017-07-07

```

Modify the `update` command too.

**Comment by sriv** [comment]
related #696

**Comment by sriv** [comment]
Fixed via #724, should be available in nightly >= 18/7/2017

**Comment by sguptatw** [comment]
Previously during installation I got the version currently installed. That is useful.
Change the message to 
Successfully installed plugin <plugin name> <version>.
